assets/src/edit-story/components/form/dropDown/list.js

```diff
@@ -210,7 +210,7 @@ DropDownList.propTypes = {
   toggleOptions: PropTypes.func.isRequired,
   handleCurrentValue: PropTypes.func.isRequired,
   value: PropTypes.string,
-  ariaLabel: PropTypes.string,
+  labelledBy: PropTypes.string,
```

Comment:
What's this doing?

Reply:
Not much, just visiting temporarily.

(Leftover of the process of fixing the dropdown label, will remove)